ci/run.yml

```diff
@@ -0,0 +1,25 @@
+- hosts: controller
+  vars:
+    fvt_dir: "{{ ansible_env.HOME}}/{{ zuul.projects['github.com/infrawatch/feature-verification-tests'].src_dir }}"
+  tasks:
+    - name: Install ansible so that ansible-playbook is available
+      ansible.builtin.pip:
+        name:
+          - ansible
+
```

Comment:
Add in a login command here. Either using rhol_crc or `command: cmd='oc login -u {{ cifmw_openshift_user }} -p {{ cifmw_openshift_password }} {{ cifmw_openshift_api }}'`

Code suggestions should be audited to update format, and make sure they're correct